### ISSUE TITLE
ipa.convert("upset" causes an error

### ISSUE THREAD

**Issue by paulvonhippel** [mphilli/English-to-IPA#34]
Running this code
```
 import eng_to_ipa as ipa
 print(ipa.convert("upset"))
```
returns 
` KeyError: 'ə'`

**Comment by mphilli** [comment]
Fixed

**Comment by paulvonhippel** [comment]
I'm still getting the same error. I ran:
`pip install git+https://github.com/mphilli/English-to-IPA`
Then in Python:
```
import eng_to_ipa as ipa
print(ipa.convert("upset"))
```
And this was the result:
`KeyError: 'ə'`

**Comment by ryanavella** [comment]
@paulvonhippel I can not replicate, I get the following:
```
əpˈsɛt
```
As a sanity check, is it possible you installed it under the wrong python toolchain? You could verify by installing via `python -m pip` instead of `pip`?

**Comment by paulvonhippel** [comment]
I installed using
python -m pip install git+https://github.com/mphilli/English-to-IPA
and got the same error....

**Comment by paulvonhippel** [comment]
How did you install?

**Comment by ryanavella** [comment]
Sorry, I noticed an issue with my attempted reproduction, and thought I deleted my comment fast enough to not trigger the GitHub notification. I thought maybe there was an issue with how I tried to reproduce it.

For what it is worth though, if I just run `python -m pip install git+https://github.com/mphilli/English-to-IPA` followed by your test-case, I don't get the `KeyError` exception you are seeing. When you install with that command, does it show it resolving to commit `82cf0f87d85dcec37996926a7a20e5b7d39397d4`?
